ns3777k/prometheus-bundle/1.0/config/routes/metrics.yaml

```diff
@@ -0,0 +1,2 @@
+metrics:
```

Comment:
Could you rename this file to `ns3777k_prometheus .yaml` or `ns3777k_prometheus_metrics.yaml`?

Reply:
yup